### ISSUE TITLE
Missing diagnostic messages to broadcast the state of the laser

### ISSUE THREAD

**Issue by pomerlef** [clearpathrobotics/LMS1xx#24]
As suggest by the [best practices](http://answers.ros.org/question/11844/when-should-i-use-diagnostic_aggregator/), all driver should publish `diagnostic_msgs/DiagnosticArray`.

**Comment by mikepurvis** [comment]
Absolutely, that'd be great. If you'd be willing to supply a PR for that, I'd be really appreciative.
